functions/_fzf_search_processes.fish

```diff
@@ -1,17 +1,19 @@
 function _fzf_search_processes --description "Search all running processes. Replace the current token with the pid of the selected process."
+    # Prefer ps binary over ps alias
+    set -f ps_cmd (command -v ps || echo "ps")
     # use all caps to be consistent with ps default format
     # snake_case because ps doesn't seem to allow spaces in the field names
     set -f ps_preview_fmt (string join ',' 'pid' 'ppid=PARENT' 'user' '%cpu' 'rss=RSS_IN_KB' 'start=START_TIME' 'command')
     set -f processes_selected (
-        ps -A -opid,command | \
+        $ps_cmd -A -opid,command | \
         _fzf_wrapper --multi \
                     --prompt="Search Processes> " \
                     --query (commandline --current-token) \
                     --ansi \
                     # first line outputted by ps is a header, so we need to mark it as so
                     --header-lines=1 \
                     # ps uses exit code 1 if the process was not found, in which case show an message explaining so
-                    --preview="ps -o '$ps_preview_fmt' -p {1} || echo 'Cannot preview {1} because it exited.'" \
+                    --preview="$ps_cmd -ww -o '$ps_preview_fmt' -p {1} || echo 'Cannot preview {1} because it exited.'" \
```

Comment:
Is -w an option available on the default ps that ships with Linux, macOS, and whatever WIndows people use (sorry I'm rusty)?

Reply:
I'm not that sure about Windows, do they have their own version of ps?

Anyway for Linux I can say that it works well on my machine (Arch Linux), and also the man page has this (https://man7.org/linux/man-pages/man1/ps.1.html).
For Mac I don't have one to try, but the man page seems to also have it (https://ss64.com/osx/ps.html).